shared/src/commonMain/graphql/schema.graphqls

```diff
@@ -86,7 +86,7 @@ type Session {
   """
   language: String
 
-  room: Room @deprecated(reason: "use rooms instead")
+  room: Room
```

Comment:
I "undeprecated" 😅  `session.room` because it was easy and `rooms` only make sense for service sessions that would somewhat be better modeled using https://github.com/joreilly/Confetti/issues/268. 
Ultimately, I think `@room` needs to go completely and be moved to a `SessionSlot` model instead.